### ISSUE TITLE
New documentIntelligenceResourceGroupLocation resource group

### ISSUE THREAD

**Issue by pope-llamas0g** [Azure-Samples/azure-search-openai-demo#1297]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [X] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> azd up with authentication enabled

### Any log messages given by the failure
>

### Expected/desired behavior
>

### OS and Version?
> macOS and devcontainers

### azd version?
1.5.1

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

Hi, I just synced my branch with the latest releases from 3 weeks ago and ran the "azd up". I noticed that a new resource was prompted to choose the location and ended with the following:

A new config file under .azure folder
![Screenshot 2024-02-21 at 11 46 59 AM](https://github.com/Azure-Samples/azure-search-openai-demo/assets/146399031/62f238bd-20fa-40f8-856e-31e0ccf55784)

I didn't get any error during deployment but wondering why I have two document Intelligence objetcs now
![Screenshot 2024-02-21 at 11 51 18 AM](https://github.com/Azure-Samples/azure-search-openai-demo/assets/146399031/f9418b64-9326-4b6d-be6c-932d0057738d)

Many thanks!

**Comment by pamelafox** [comment]
@pope-llamas0g Good question, I discuss it here: https://github.com/Azure-Samples/azure-search-openai-demo/releases/tag/2024-02-15b
You may delete the old one.

**Comment by pope-llamas0g** [comment]
Hi @pamelafox thank you so much for your swift response. I am sorry I didn't read the release notes.

So, I can just delete the one from Switzerland North on the azure portal since that is not a supported region, right? Just double-checking as I am afraid of breaking the system.

**Comment by pamelafox** [comment]
@pope-llamas0g Yes, you can delete the old one that has "fr" in the name. We don't actually use Document Intelligence in the deployed user-facing app, so the only system you could break would be scripts/prepdoc.sh. You can re-run scripts/prepdoc.sh after to make sure all is working as intended, if you're nervous.
